tools/mpremote/mpremote/commands.py

```diff
@@ -375,6 +394,8 @@ def do_filesystem(state, args):
         raise CommandError("{}: {}: File exists.".format(command, er.args[0]))
     except TransportError as er:
         raise CommandError("Error with transport:\n{}".format(er.args[0]))
+    except OSError as er:
+        raise CommandError(f"Error: {er}") from er
```

Comment:
This should match the other lines above in coding style, eg:
```py
raise CommandError("Error: {} {}".format(er.args[0], er.args[1])
```

Reply:
I changed the format as that lead to a too verbose error message repeating the same information multiple times.
With the updated logic the OSErrors are caught and handlerd deeper in `do_filesystem_recursive_rm` so this clause is no longer needed.